sentry_sdk/integrations/aiohttp.py

```diff
@@ -141,7 +141,7 @@ async def sentry_app_handle(self, request, *args, **kwargs):
                         transaction.set_http_status(response.status)
                         return response
 
-        Application._handle = sentry_app_handle  # type: ignore[method-assign]
```

Comment:
`mypy` was complaining about the `type: ignore` comments in this file, so I had to make changes here, even though the changes are unrelated to the rest of the PR